src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/resource/BulkAlertsController.kt

```diff
@@ -1,44 +1,39 @@
 package uk.gov.justice.digital.hmpps.hmppsalertsapi.resource
 
 import io.swagger.v3.oas.annotations.Operation
-import io.swagger.v3.oas.annotations.Parameter
 import io.swagger.v3.oas.annotations.media.Content
 import io.swagger.v3.oas.annotations.media.Schema
 import io.swagger.v3.oas.annotations.responses.ApiResponse
 import io.swagger.v3.oas.annotations.responses.ApiResponses
-import jakarta.servlet.http.HttpServletRequest
-import jakarta.validation.Valid
 import org.springframework.http.HttpStatus
 import org.springframework.http.MediaType
 import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.PatchMapping
+import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.ResponseStatus
 import org.springframework.web.bind.annotation.RestController
-import uk.gov.justice.digital.hmpps.hmppsalertsapi.config.AlertRequestContext
-import uk.gov.justice.digital.hmpps.hmppsalertsapi.model.BulkAlert
 import uk.gov.justice.digital.hmpps.hmppsalertsapi.model.BulkAlertPlan
-import uk.gov.justice.digital.hmpps.hmppsalertsapi.model.request.BulkCreateAlerts
-import uk.gov.justice.digital.hmpps.hmppsalertsapi.service.BulkAlertService
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.model.BulkPlan
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.model.request.BulkRequest
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.service.PlanBulkAlert
 import uk.gov.justice.hmpps.kotlin.common.ErrorResponse
+import java.util.UUID
 
 @RestController
 @RequestMapping("/bulk-alerts", produces = [MediaType.APPLICATION_JSON_VALUE])
-class BulkAlertsController(
-  private val bulkAlertService: BulkAlertService,
-) {
-  @ResponseStatus(HttpStatus.CREATED)
-  @PostMapping
-  @Operation(
-    summary = "Create alerts for multiple people in bulk",
-  )
+class BulkAlertsController(private val plan: PlanBulkAlert) {
+
+  @PostMapping("/plan")
+  @Operation(summary = "Create the plan for bulk alerts")
```

Comment:
i expected it to be a separate new endpoint, and the old endpoint to be deprecated later.
but Joe HD cfm'd that it's ok for bulk alerts to stop working for dev today, so replacing the old endpoint is even better!